pages/common/xml-edit.md

```diff
@@ -0,0 +1,28 @@
+# xml edit
+
+> Edit an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Delete elements matching an XPATH from an XML document:
+
+`xml edit --delete {{"XPATH1"}} {{path/to/input.xml|URI}}`
+
+- Move an element node of an XML document from XPATH1 to XPATH2:
+
+`xml edit --move {{"XPATH1"}} {{"XPATH2"}} {{path/to/input.xml|URI}}`
+
+- Rename all attributes named "id" to "ID":
+
+`xml edit --rename {{"//*/@id"}} -v {{"ID"}} {{path/to/input.xml|URI}}`
+
+- Rename XML elements of "table" named "rec" to "record":
```

Comment:
What is `table` here exactly? Is it a node?

Reply:
Yes, its an XML node name. The example is from the manual. It presumes a table called "table" with records called "rec" that get renamed to "record". The manual provides the contents of the data file that it operates on but there is no place for that here.

Reply:
How about if we just add
`Rename XML elements of the node "table" named "rec" to "record"`?

Reply:
- Rename sub-elements of the element "table" that are named "rec" to "record":